### ISSUE TITLE
Change the way the tournaments are sorted on a profile page to by completed date

### ISSUE THREAD

**Issue by danbugs** [danbugs/smithereens#107]
**Describe the solution you'd like**
GENESIS X was this past weekend, but, still, it doesn't show in MkLeo's first page (only on page 6). This is probably because the event for GENESIS X was created way earlier and our current sorting goes off that.

**Additional context**
n/a

**Comment by DhruvrajsinhRathod** [comment]
Can I try working on this issue?

**Comment by danbugs** [comment]
Yes, @DhruvrajsinhRathod - please go ahead! I'll assign it to you! Are you in our Discord server? We have a dev channel there for discussions in case you have any questions :) 

Here's a link invite if not: https://discord.gg/6B3D2MDmnn

**Comment by danbugs** [comment]
@DhruvrajsinhRathod ~ Are you still interested in working on this feature? No worries, if not.

**Comment by danbugs** [comment]
No longer applicable w/ V2 (see #135)
